src/app/response-status-bar.component.html

```diff
@@ -7,15 +7,17 @@
                         <i class="ms-Icon" [ngClass]="message?.icon"></i>
                     </div>
                 </td>
-                    <div class="ms-MessageBar-actionables">
-                        <div class="ms-MessageBar-text" [innerHtml]="messageHTML"></div>
-                    </div>
+                <div class="ms-MessageBar-actionables">
+                    <div class="ms-MessageBar-text" tabindex="0" [innerHtml]="messageHTML"></div>
+                </div>
                 <td>
                 </td>
                 <td>
                     <div class="ms-MessageBar-actionsOneline">
                         <div id="dismiss-btn" class="ms-MessageBar-icon">
-                            <a href="#" (click)="clearMessage()"><i class="ms-Icon ms-Icon--Cancel"></i></a>
+                            <div tabindex="0" role="button" aria-label="Clear message" (click)="clearMessage()">
```

Comment:
Would "Dismiss message" make more sense here? I'm guessing this is for the popup that shows the response status code.

Reply:
Good catch. Dismiss or close both make more sense than clear. i'm thinking that close is a bit more concise than dismiss. I'll change the label to "Close status message"